neighborDist: 10.0
maxNeighbors: 10
timeHorizon: 8.0
timeHorizonObst: 8.0
radius: 0.19
maxSpeed: 0.10
timestep: 0.05
axel_width: 0.23

waypoint_spacing: 0.75
path_margin: 0.3
goal_tolerance: 0.15
los_margin: 0.1
